examples/cp-all-in-one/README.md

```diff
@@ -0,0 +1,6 @@
+![image](../confluent-logo-300-2.png)
+
+# Overview
+
+This `docker-compose.yml` includes all the components of the Confluent Platform to demonstrate how they can be configured to interoperate.
+For an example of how to use this Docker setup, refer to the Confluent Platform quickstart: https://docs.confluent.io/current/quickstart/index.html
```

Comment:
```suggestion
For an example of how to use this Docker setup, refer to the :ref:`quickstart`.
```

Reply:
This isn't docs, so the ref won't resolve